docs/docs/smart-contracts/PSP22/Extensions/wrapper.md

```diff
@@ -0,0 +1,61 @@
+---
+sidebar_position: 4
+title: PSP22 Wrapper
+---
+
+This example shows how you can reuse the implementation of [PSP22](https://github.com/Supercolony-net/openbrush-contracts/tree/main/contracts/token/psp22) token with [PSP22 Wrapper](https://github.com/Supercolony-net/openbrush-contracts/tree/main/contracts/token/psp22/extensions/wrapper) extension.
```

Comment:
Could you describe a little more the idea of that extension for the user pelase?=)